### PR TITLE
BAP-54 table binding modified

### PR DESCRIPTION
**Pull request by PavelAAlexeevQ**

### CODE REVIEW

**Review by SergeyRuzin — DISMISSED** [review]

File: src/Behavioral.Automation/Bindings/TableBinding.cs [hunk 1]
```diff
@@ -20,9 +20,9 @@ public void CheckRowsCount([NotNull] ITableWrapper element, int count)
             Assert.ShouldBecome(() => element.Rows.Count(), count, $"{element.Caption} has {element.Rows.Count()} rows");
         }
 
-        [Given("(.*?) (contain|not contain) the following rows:")]
-        [Then("(.*?) should (contain|contain in exact order|not contain) the following rows:")]
-        public void CheckTableContainsRows(ITableWrapper gridRows, string behavior, Table table)
+        [Given("(.*?) (contain|not contain) the (following|only following) rows:")]
```
Comment (SergeyRuzin) [inline src/Behavioral.Automation/Bindings/TableBinding.cs:23]: Just as a suggestion. Maybe "the following (rows | rows only) will sound better. Don't you think?

Comment (PavelAAlexeevQ) [reply src/Behavioral.Automation/Bindings/TableBinding.cs:23]: 100% agree )

**Review by mikhail1aleksandrov1 — DISMISSED** [review]

File: src/Behavioral.Automation/Bindings/TableBinding.cs [hunk 2]
```diff
@@ -42,6 +42,12 @@ public void CheckTableContainsRows(ITableWrapper gridRows, string behavior, Tabl
                     true,
                     $"{gridRows.Caption} is {gridRows.CellsText.Aggregate((x, y) => $"{x}, {y}")}");
             }
+            
+            if(strictCondition == "only following")
```
Comment (SergeyRuzin) [inline src/Behavioral.Automation/Bindings/TableBinding.cs:46]: Shouldn't it be "rows only"?

**Review by SergeyRuzin — APPROVED** [review]

**Review by SergeyRuzin — APPROVED** [review]

File: src/Behavioral.Automation/Bindings/TableBinding.cs [hunk 3]
```diff
@@ -38,10 +38,15 @@ public void CheckTableContainsRows(ITableWrapper gridRows, string behavior, Tabl
             }
             else
             {
+                if (strictCondition == "rows only")
+                {
+                    Assert.ShouldBecome(() => gridRows.CellsText.Count() == expectedValues.Count, true,
+                        $"{gridRows.Caption} is {gridRows.CellsText.Aggregate((x, y) => $"{x}, {y}")}");
```
Comment (afomina-quantori) [inline src/Behavioral.Automation/Bindings/TableBinding.cs:44]: I would expect to see the message like this:
"Expected number of rows is ..., but actual is ..."

**Review by afomina-quantori — APPROVED** [review]

**Pull request merged**